pandas/io/excel/_base.py

```diff
@@ -366,6 +366,9 @@ def _workbook_class(self):
     def load_workbook(self, filepath_or_buffer):
         pass
 
+    def close(self):
```

Comment:
Would it make sense to make this an abstract method, and make child classes be explicit on how the close is performed (including when no close is performed)?

Reply:
In an earlier version of this PR, this was an abstract method, but the powers that be preferred this.

Reply:
We can't make a new abstract method in a bugfix release. Any child not implementing it would fail when the class is instantiated.